pkg/appconsts/appconsts.go

```diff
@@ -72,6 +72,13 @@ const (
 	// data square.
 	MinShareCount = DefaultMinSquareSize * DefaultMinSquareSize
 
+	// SubtreeRootHeightThreshold dictates the threashold for increasing the sub
+	// tree root height for blobs. If the number of subtree roots used to create
+	// a share commitment surpasses this threshold, then the height is
+	// increased. The rationale for this value is described in more detail in
+	// ADR013 (./docs/architecture/adr-013).
+	SubtreeRootHeightThreshold = 128
```

Comment:
[question] Does `height` mean the actual tree height or does it mean the total number of sub-tree roots? The term `height` is not used in the ADR 13.

Reply:
height is the height of the subtree, so if 128 subtree roots are crossed, then the height of the subtree is increased

you are definitely correct that the ADR does not mention height! One thing that I want to do in https://github.com/celestiaorg/celestia-app/issues/1607 is to make a clearer reference to the algorithm, as I don't believe the part where we increase the height (aka round the sub tree width up to a power of two) is clear enough

per the seventh paragraph in the Context section
> ...Blobs that are larger than MaxSquareSize * 2 but smaller than MaxSquareSize * 4 can use the index of 4 to get a maximum of 3 padding shares and so on.

I interpreted this to mean that we are using a higher sub tree root

Reply:
I tend to agree with omitting height from the name

Reply:
Thanks for clarification on this. I have a followup question for @nashqueue:
Quoting from ADR 13, where there is an argument on the proof size and its relation with the number of subtree roots for a given blob, i.e., 
>Small blob sizes have the lowest ratio but also small inclusion proofs. Therefore increasing the proof size is not a problem until some threshold. It would increase the ratio of blob size to padding for small blobs which have the worst ratio.
 Assuming there is a threshold, the number of subtree roots in a proof, where the proof size of a blob is acceptable, we can use this threshold to determine the index of the blob in the square. This would give us zero padding for blobs that are smaller than the threshold and non-zero padding for blobs that are larger than the threshold but still smaller than before.

My understanding is that the `SubtreeRootThreshold` in the code corresponds to the threshold mentioned in the paragraph above. So, my question is that what is this threshold mean to enforce, if any? Does this threshold is to enforce a hard cap on the number of subtree roots per blob (as the current algorithm stands, the number of subtree roots per blob is not capped, take a blob share size of 72 and a threshold of 8 as a counterexample).